tsconfig.base.json

```diff
@@ -1,21 +1,8 @@
 {
+  "extends": "@qlik/tsconfig/react.json",
   "compilerOptions": {
-    "outDir": "lib",
-    "module": "ESNext",
-    "target": "ES2022",
-    "sourceMap": true,
-    "jsx": "react",
-    "declaration": true,
-    "isolatedModules": true,
-    "strict": true,
-  
-    "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
-    "resolveJsonModule": true,
-    "forceConsistentCasingInFileNames": true,
-    "skipLibCheck": true,
-    "preserveSymlinks": true
+    "jsx": "react"
+    // "noUncheckedIndexedAccess": true
```

Comment:
This one is the `@qlik/tsconfig/react.json`:
<img width="556" alt="Screenshot 2023-08-23 at 09 33 23" src="https://github.com/qlik-oss/nebula-table-utils/assets/29652890/239f0fd2-18b7-438e-b68e-c51c37f9dd9f">

There are some stuff missing like "noEmit", "declaration" and "outDir" is that my eyes catched them. These were important from the pkg perspective. for example we need declaration files for types, and ts outDir defaults to '<rootDir>/dist'

Reply:
`noEmit` is there but I will add `declartion` and `outDir`